trader/trader.go

```diff
@@ -443,24 +493,54 @@ func (t *Trader) update() bool {
 				t.deleteAllOffers(true)
 			}
 		})
+		if e != nil {
+			log.Println(e)
+			t.deleteAllOffers(false)
+			return plugins.UpdateLoopResult{
+				Success:            false,
+				NumPruneOps:        numPruneOps,
+				NumUpdateOpsDelete: numUpdateOpsDelete,
+				NumUpdateOpsUpdate: numUpdateOpsUpdate,
+				NumUpdateOpsCreate: numUpdateOpsCreate,
+			}
+		}
 
+		numUpdateOpsCreate, numUpdateOpsDelete, numUpdateOpsUpdate, e = t.countOfferChangeTypes(ops)
```

Comment:
we should run this on the `opsOld` value (i.e. after we call `t.strategy.UpdateWithOps(t.buyingAOffers, t.sellingAOffers`) instead of here.